lib/licensee/project.rb

```diff
@@ -2,17 +2,6 @@ class Licensee
   class Project
     attr_reader :repository
 
-    # Array of file names to look for potential license files, in order
-    # Filenames should be lower case as candidates are downcased before comparison
-    LICENSE_FILENAMES = %w[
```

Comment:
The backwards-incompatible consequence of this change is that the license filenames are not picked by the order of preference anymore. Now, the "COPYING" file will win over a "LICENSE" file in the same repo, when previously the opposite was the case.